src/commands/lambda/flare.ts

```diff
@@ -253,8 +261,9 @@ export class LambdaFlareCommand extends Command {
       await zipContents(rootFolderPath, zipPath)
 
       // Send to Datadog
-      this.context.stdout.write('\n🚀 Sending to Datadog Support...\n')
-      await sendToDatadog(zipPath, this.caseId!, this.email!, this.apiKey!, rootFolderPath)
+      const endpointUrl = getEndpointUrl()
+      this.context.stdout.write(`\n🚀 Sending to Datadog Support via ${endpointUrl}...\n`)
```

Comment:
This information is not relevant to the user.
```suggestion
      this.context.stdout.write('\n🚀 Sending to Datadog Support...\n')
```